src/components/pulp-code-block.tsx

```diff
@@ -0,0 +1,28 @@
+import {CodeBlock, CodeBlockAction, CodeBlockCode, ExpandableSection} from "@patternfly/react-core";
+import {PulpCopyButton} from "src/components/pulp-copy-button";
+import {PulpDownloadButton} from "src/components/pulp-download-button";
+import {t} from "@lingui/macro";
+import React from "react";
+
+export const PulpCodeBlock = ({ code, filename }: { code: string, filename?: string }) => {
+  const name = filename ? filename : 'pulpui-codeblock-download';
+  const actions = (
+    <CodeBlockAction>
+      <PulpDownloadButton text={code} filename={name} />
+      <PulpCopyButton text={code} textId='code-content' />
```

Comment:
can we remove the hardcoded id? It can't really work with multiple codeblocks on one screen.